qa/os/src/test/java/org/elasticsearch/packaging/test/ArchiveTests.java

```diff
@@ -111,19 +121,10 @@ public void test50StartAndStop() throws Exception {
         // cleanup from previous test
```

Comment:
Before this test (`test50StartAndStop`) I would suggest we try different ways (using randomization or not), in which the node doesn't start with TLS auto-configured, because of node roles or discovery settings.

Reply:
Added such test cases